mne_bids/write.py

```diff
@@ -637,6 +637,23 @@ def _sidecar_json(raw, task, manufacturer, fname, datatype, overwrite=False,
     n_stimchan = len([ch for ch in raw.info['chs']
                       if ch['kind'] == FIFF.FIFFV_STIM_CH]) - n_ignored
 
+    # for Neuromag MEG files, check whether Extra points are present
+    # and set DigitizedHeadPoints accordingly, and whether RPA/LPA/
+    # NAS is present, and set DigitizedLandmarks accordingly
+    digitized_head_points = False
+    digitized_landmark = False
+    if datatype == 'meg':
+        if raw.info['dig'] is None:
+            raw.info['dig'] = []
```

Comment:
let's not modify raw.info['dig']

```suggestion
    if datatype == 'meg' and raw.info['dig'] is not None:
```